source/assets/sass/components/_buttons.scss

```diff
@@ -1,45 +1,59 @@
 @use 'sass:color';
+@use '../visual-design/typography';
```

Comment:
Consider splitting `visual-design/typography` into a typography config (with the things that get used in various places) and a typography partial that outputs CSS, and only needs to be used in the final output?

Reply:
Added this suggestion to the card: https://trello.com/c/V2ujZ5Pc/5-split-config-only-css-from-the-css-that-has-output

Reply:
done in https://github.com/oddbird/sass-site/pull/58/commits/31d9b56e1e9cc6e0ff132c926354dc7f6f82bb2e